tests/test_warning.py

```diff
@@ -0,0 +1,25 @@
+# import posthog
+
+from unittest.mock import Mock
+from ploomber_core.warnings import deprecation_warning
+from ploomber_core.telemetry import telemetry as telemetry_module
+
+import pytest
+
+
+def test_deprecation_warning_w_posthug(monkeypatch):
```

Comment:
```suggestion
def test_deprecation_warning_w_posthog(monkeypatch):
```

Reply:
fix in [557b5ad](https://github.com/ploomber/core/pull/65/commits/557b5ad96968747255a5006d186873e84d2e37b8)